packages/widgetbook/lib/src/knobs/builders/double_knobs_builder.dart

```diff
@@ -17,17 +17,30 @@ class DoubleKnobsBuilder {
     double? min,
     int? divisions,
   }) {
-    initialValue ??= max ?? min ?? 10;
-    return onKnobAdded(
-      DoubleSliderKnob(
-        label: label,
-        value: initialValue,
-        description: description,
-        min: min ?? initialValue - 10,
-        max: max ?? initialValue + 10,
-        divisions: divisions,
-      ),
-    )!;
+    if (initialValue == null) {
+      return onKnobAdded(
+        DoubleSliderKnob(
+          label: label,
+          value: initialValue,
+          description: description,
+          min: min!,
+          max: max!,
+          divisions: divisions,
+        ),
+      )!;
+    } else {
+      initialValue = max ?? min ?? 10;
+      return onKnobAdded(
+        DoubleSliderKnob(
+          label: label,
+          value: initialValue,
+          description: description,
+          min: min ?? initialValue - 10,
+          max: max ?? initialValue + 10,
+          divisions: divisions,
+        ),
+      )!;
+    }
```

Comment:
This change will be **breaking** for all users who have an `initialValue` of null without `min` and `max` values set, due to the null assertion on `min!` and `max!`

Consider the following change instead:
```suggestion
    return onKnobAdded(
      DoubleSliderKnob.nullable(
        label: label,
        value: initialValue,
        description: description,
        min: min ?? (initialValue == null ? 0 : initialValue - 10),
        max: max ?? (initialValue == null ? 10 : initialValue + 10),
        divisions: divisions,
      ),
    );
```